Content.Server/NPC/Components/FactionExceptionComponent.cs

```diff
@@ -6,7 +6,7 @@ namespace Content.Server.NPC.Components;
 /// Prevents an NPC from attacking ignored entities from enemy factions.
 /// Can be added to if pettable, see PettableFriendComponent.
 /// </summary>
-[RegisterComponent, Access(typeof(NpcFactionSystem))]
+[RegisterComponent, Access(typeof(NpcFactionSystem), typeof(NPCImpritingBehaviourSystem))]
 public sealed partial class FactionExceptionComponent : Component
 {
```

Comment:
If we start adding a billion access overrides they should be going through a system as then there's no reason to have access, but I'll fix this when I rework npcs.

Reply:
added To Do